### PR TITLE
Created new adaptor for Rivera Money

### PR DESCRIPTION
**Pull request by GausPeerzade**
I have created a new yield adaptor with all the currently live rivera money erc4626 vaults ,vault addresses with chain are in config object and due to some installation and local machine issue testing of this adaptor was not working so i am just going to push the adaptor with all detailed required hoping someone from the team will help me out with this as soon as possible.Thank you

### CODE REVIEW

File: src/adaptors/rivera-money/index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+const { getERC4626Info } = require('../../helper/erc4626');
```
Comment (slasher125) [inline src/adaptors/rivera-money/index.js:1]: can remove this line, the function is now accessible via `utils.getERC4626Info`

File: src/adaptors/rivera-money/index.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    core: [
+        "0xBC91a7a0eE37085af193C61747ecE693979Ec0C1",
+        "0x018BeE125A17D456E6dacE22A66E8B9aF3c69449",
+    ],
+}
+
+
+const poolsFunction = async () => {
+    Object.keys(config).forEach(chain => {
```
Comment (slasher125) [inline src/adaptors/rivera-money/index.js:64]: refactor to for const loop and return the full pool response

File: src/adaptors/rivera-money/index.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        const vaults = config[chain]
+        for (let i = 0; i < vaults.length; i++) {
+            const data = getERC4626Info(vaults[i], chain);
```
Comment (slasher125) [inline src/adaptors/rivera-money/index.js:67]: await

**Comment by slasher125** [comment]
@GausPeerzade left some comments

**Comment by GausPeerzade** [comment]
sorry i was on holiday , will look into it tomorrow

**Comment by slasher125** [comment]
gm @GausPeerzade are u still on this?

**Pull request closed without merging**